src/upgrade.js

```diff
@@ -52,8 +52,8 @@ available updates.\
     getInstalledPackages(options) {
       let packages = [];
       for (let name of fs.list(this.atomPackagesDirectory)) {
-        var pack;
-        if (pack = this.getIntalledPackage(name)) {
+        let pack = this.getInstalledPackage(name);
```

Comment:
```suggestion
        let pack = this.getIntalledPackage(name);
```

This makes tests pass again for me.

Reply:
Thanks for the diff @DeeDeeG, and thanks for staying more dedicated to my PR than I've apparently been lol. I'll get this one merged and at a later time deal with the typo

Reply:
No worries, it keeps showing in my GitHub inbox, that's how I keep noticing it's pending yet. Very close to the finish line so I thought I'd bop in and post a diff.